rpc_jobs/rpc_octavia_gating.yml

```diff
@@ -0,0 +1,46 @@
+ - project:
+    name: "rpc-octavia"
+
+    # This is the URL for the project's repository
```

Comment:
nit: You can remove all the comments and just use your own. The comments are in the documentation anyway.

Reply:
+1